test/reducers/reducerTest.js

```diff
@@ -203,26 +170,10 @@ describe('reducers', () => {
                    'selected task should not have archivedAt')
       assert.equal(dateStr, actual[1].archivedAt,
                    'deselected task should have archivedAt')
-
-      const archivedTaskKeys = storage.get('archived')
-      assert(archivedTaskKeys.indexOf('issue-99') < 0,
-             'task key should not be in the archived list')
```

Comment:
This test still needs some check to ensure what was once archived is now restored. Maybe after the restore step, you assert that archivedAt is now null on the task?